src/base/bittorrent/customstorage.cpp

```diff
@@ -35,6 +35,190 @@
 #include "base/utils/fs.h"
 #include "common.h"
 
+#if (LIBTORRENT_VERSION_NUM >= 20000)
+#include <libtorrent/session.hpp>
```

Comment:
Likewise, here it would become:
```c++
#include "customstorage.h"

#include <libtorrent/download_priority.hpp>

#if (LIBTORRENT_VERSION_NUM >= 20000)
#include <libtorrent/session.hpp>
#endif

#include <QDir>

#include "base/utils/fs.h"
#include "common.h"
```